### PR TITLE
different numa nodes

### PR DESCRIPTION
**Pull request by NHoff95**

### CODE REVIEW

File: src/TableGenerator.cpp [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
+
+Table TableGenerator::generateTable(unsigned int numOfLocalColumns, unsigned int numOfRemoteColumns,
+                                    unsigned long numOfRows, unsigned int maxRandomNumberInCell) {
+    srand(time(NULL));
+    Table table;
+    for (unsigned int i = 0; i < numOfLocalColumns; ++i) {
+        addColumn(numOfRows, maxRandomNumberInCell, 0, table);
+    }
+    for (unsigned int i = 0; i < numOfRemoteColumns; ++i) {
+        addColumn(numOfRows, maxRandomNumberInCell, (((int) Random::next()) % (numa_max_node() + 1)), table);
```
Comment (StarkRobert) [inline src/TableGenerator.cpp]: change to:
fixed: (((int) Random::next()) % (numa_max_node()))) + 1
so we do not get 0 (local node)

**Review by StarkRobert — COMMENTED** [review]
Everything else seems great!

File: src/TableGenerator.cpp (hunk 1 above)
Comment (lawben) [reply src/TableGenerator.cpp]: Not quite, it should be `(Random::next() % numa_max_node()) + 1`
The +1 in the `numa_max_node` would give us 0 to 3 and the +1 at the end could give us a 4. So we want the range (0 to 2) and then +1 to give us 1 to 3, which are all valid and not local.

**Comment by WGierke** [comment]
I'm getting a 
```7067 floating point exception (core dumped)  ./src/tuk_numa_benchmark --benchmark_format=json```  (16 GB RAM)

**Review by lawben — APPROVED** [review]
LGTM

**Pull request merged**